### DISCUSSION TITLE
Add Minimum Higher Bid when creating NFT Auction

### DISCUSSION THREAD

**Discussion by brootle** [deso-protocol/dips#156] [Medium CIPs]
The title says it all, add minimum higher bid option for NFT auctions.

Read more about motivation in @milhousevanhouten post at BitClout https://bitclout.com/posts/eb00dc92524a829aa1a076eaa0a7e07266eeec48594861af4c409e9eea866f19

I copy the post below for the reference 👇

> A bidding war style auction with no set end date is my least favorite auction to be involved in. I don't enjoy watching the auctions so closely and worry I'll lose to .01 more than I bid while I'm sleeping.
> 
> It's good for the artist on that first few auctions, but I think over time buyers get annoyed with that in the same way I do. Also there's only 20 or 30 people seriously bidding on these auctions, so you're bidding .01 against the same people over and over again. It's asinine.
> 
> Sent quite a few "hey do you really love this one?" or "I really love this one can I have it?" messages haha. I don't want to bid against the homies but what can you do?
> 
> @brootle write us up a "minimum higher bid" function for artists to set when they create an auction!

**Comment by WhilM** [comment]
This single change should help buyers and artists on bitclout. Please consider prioritizing this higher in the future. Thanks!
